### PR TITLE
TempusDominus is updated and improved

### PR DESCRIPTION
**Pull request by solomax**

### CODE REVIEW

File: bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/form/tempusdominus/TempusDominusConfig.java [hunk 1]
```diff
@@ -260,8 +261,8 @@ public TempusDominusConfig withViewDate(Date date) {
     /**
      * @return Date format pattern set for TempusDominusLocalizationConfig.getFormat()
      */
-    public String getFormat() {
-        return localization.getFormat();
+    public String getJavaFormat() {
```
Comment (martin-g) [inline bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/form/tempusdominus/TempusDominusConfig.java:277]: Can we keep the old name ? For backward compatibility
E.g. `getFomat()` could be `@Deprecated` and delegating to `getJavaFormat()`

Comment (solomax) [reply bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/form/tempusdominus/TempusDominusConfig.java:277]: done!
Thanks for the review :)

**Comment by martin-g** [comment]
@solomax The backport to wicket-9.x-bootstrap-5.x fails due to the usage of `"""..."""` strings in the samples page.

**Comment by solomax** [comment]
@martin-g could you please deploy SNAPSHOT with these changes? :)

**Comment by solomax** [comment]
> @solomax The backport to wicket-9.x-bootstrap-5.x fails due to the usage of `"""..."""` strings in the samples page.

I'll create separate PR :)

**Comment by martin-g** [comment]
OK!

**Comment by martin-g** [comment]
> @martin-g could you please deploy SNAPSHOT with these changes? :)

Done!

**Pull request merged**